### PR TITLE
Adding my Coordinates 

### PR DESCRIPTION
**Pull request by gharam37**
Test

### CODE REVIEW

**Comment by to-dy** [comment]
@gharam37 

Add the three dashes, and you should be good to go!
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

File: pins [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: gharam
```
Comment (obaceski) [inline pins]: I think you are missing the 3 dashes

File: pins [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+latitude: 30
+longitude: 70
```
Comment (obaceski) [inline pins]: again, missing 3 dashes at the end

**Comment by richardrubert** [comment]
motivation and support... go you

**Comment by quinmixon** [comment]
@gharam37 keep up the good work 👍

**Comment by brianamarie** [comment]
Hi @gharam37, everything that's been mentioned is correct! Line one should only be the characters `---`, and the last line should be, too. Let me know if you need any help!

**Comment by brianamarie** [comment]
Hi @gharam37! I went ahead and made some changes. Since it's been open for a while, I'm going to merge, too. Nice work!

**Pull request merged**